lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
Sounds good to me.

Alternatively node-pre-gyp's [`binary.find()`](https://github.com/mapbox/node-pre-gyp/blob/master/lib/pre-binding.js#L15) could be modified to accept a JSON object.

Then the code could just be:

```javascript
var bindings = require('node-pre-gyp').find(require('../package.json'));